crates/swc_ecma_minifier/tests/terser/compress/evaluate/pow_spec/output.terser.js

```diff
@@ -0,0 +1 @@
+console.log(1);
```

Comment:
@kdy1 Is there a way to update the output.terser.js ?

Reply:
You may update it manually, but being better than `output.terser.js` is fine.

Another way to update `output.terser.js` is replacing `compress.js` in terser test suite with https://github.com/swc-project/swc/blob/181320a5554a1b29a64650bdc2c1097b67fe6e9a/crates/swc_ecma_minifier/scripts/x/terser/compress.js and running the terser test.